### PR TITLE
[DUOS-2658] Validation Bug in Data Submission Form

### PR DESCRIPTION
**Pull request by hams7504**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2658

### Summary
- Commented out the code causing the error

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/assets/schemas/DataRegistrationV1Validation.js [hunk 1]
```diff
@@ -1 +1,2176 @@
...
+          const err46 = { instancePath: instancePath + "/url", schemaPath: "#/properties/url/minLength", keyword: "minLength", params: { limit: 1 }, message: "must NOT have fewer than 1 characters" };
+          if (vErrors === null) {
+            vErrors = [err46];
+          }
+          else {
+            vErrors.push(err46);
+          }
+          errors++;
+        }
+        // Default ajv URI validation is failing with "TypeError: formats12 is not a function"
```
Comment (bstitson-broad) [inline src/assets/schemas/DataRegistrationV1Validation.js:776]: @hams7504 is this the section you commented out?  I found it by searching for "//" in the diff.

Comment (hams7504) [reply src/assets/schemas/DataRegistrationV1Validation.js:776]: Yes, that's right! I added line 776 to clarify why I commented out this section.

**Review by rushtong — APPROVED** [review]
👍🏽

**Review by nmalfroy — APPROVED** [review]
I think this is ok?  Seems fairly mechanical

**Pull request merged**